ear-production-suite-plugins/plugins/direct_speakers/src/direct_speakers_plugin_processor.cpp

```diff
@@ -24,7 +24,7 @@ DirectSpeakersAudioProcessor::DirectSpeakersAudioProcessor()
   addParameter(routing_ =
     new ui::NonAutomatedParameter<AudioParameterInt>(
       "routing", "Routing",
-      -1, 63, -1));
+      -1, MAX_DAW_CHANNELS-1, -1));
```

Comment:
Works going from v6 to v7 because we sort the actual value rather than normalised. From v7 to v6 it works so long as the stored value is <=64. If it's greater, you get some funny behaviour but REAPER already warns that the project might be incompatible which I think is probably sufficient.